tsc/process/deprecation.md

```diff
@@ -0,0 +1,21 @@
+**Deprecation Strategy for OpenVDB**
+
+OpenVDB is committed to supporting the current and previous two years of the
+VFX Reference Platform.
+
+For example, as of writing it is March 2019 which means VFX Reference Platform
+2019, 2018 and 2017
+
+This infers the following support:
+
+* OpenVDB ABI=4, ABI=5 and ABI=6
+* C++11 and C++14
+* Houdini 16.5, 17.0 and 17.5
+
+Although Houdini versions are not included in the VFX Reference Platform, each
```

Comment:
Technically, we support Maya as well (though I haven't tried building for Maya 2019 yet; has anyone?), so perhaps we could collapse this into the above as follows:
```
We are committed to supporting the current year and previous two years of the
[VFX Reference Platform](http://www.vfxplatform.com/)
and all releases of Houdini and Maya based on those versions of the platform.

Reply:
Your version is better, I've updated the wording here.